live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/event/UpdateNodeEventMapper.java

```diff
@@ -61,4 +68,50 @@ public UpdateNodeMetadataEvent map(IngestNewNodeEvent event)
         node.properties().forEach(updateMetadataEvent::set);
         return updateMetadataEvent;
     }
+
+    public UpdateNodeMetadataEvent map(UpdateNodeEvent event)
+    {
+        UpdateNodeMetadataEvent updateMetadataEvent = UpdateNodeMetadataEvent.create();
+
+        applyFieldDelta(updateMetadataEvent, NAME, event.name());
+        applyFieldDelta(updateMetadataEvent, PRIMARY_ASSOC_Q_NAME, event.primaryAssocQName());
+        applyFieldDelta(updateMetadataEvent, TYPE, event.nodeType());
+        applyFieldDelta(updateMetadataEvent, MODIFIED_BY_USER_WITH_ID, event.modifiedByUserWithId());
```

Comment:
Nice thought - the "created" properties shouldn't be able to change. It might be worth testing this in case it's actually possible to retroactively set them.